packages/atomic/src/components/commerce/atomic-commerce-products-per-page/e2e/atomic-commerce-products-per-page.e2e.ts

```diff
@@ -0,0 +1,80 @@
+import {test, expect} from './fixture';
+
+test.describe('atomic-commerce-products-per-page', () => {
+  test.describe('with default parameters', () => {
+    test.beforeEach(async ({productsPerPage}) => {
+      await productsPerPage.load(undefined, 'in-page');
+    });
+
+    test('should be A11y compliant', async ({
+      productsPerPage,
+      makeAxeBuilder,
+    }) => {
+      await productsPerPage.hydrated.waitFor();
+      const accessibilityResults = await makeAxeBuilder().analyze();
+      expect(accessibilityResults.violations).toEqual([]);
+    });
+
+    test('should execute the first query with the default number of products', async ({
+      querySummary,
+    }) => {
+      await expect(
+        querySummary
+          .querySummary({indexOfFirstResult: 1, indexOfLastResults: 10})
+          .first()
+      ).toBeVisible();
+    });
+
+    test.describe('when selecting a different number of products', () => {
+      test.beforeEach(async ({productsPerPage}) => {
+        await productsPerPage.choice(25).click();
+      });
+
+      test('should render the component with the right choice selected', async ({
+        productsPerPage,
```

Comment:
I don't think that is relevant in that component since there is nothing in the logic referring to either a listing page or a search page. If there is a problem here due to it's commerce-interface type, there would be a problem in every other component.